hamster_gtk/hamster_gtk.py

```diff
@@ -275,6 +276,8 @@ def _get_default_config(self):
             'tmpfile_path': os.path.join(appdirs.user_data_dir, 'hamster-gtk.tmp'),
             'db_engine': 'sqlite',
             'db_path': os.path.join(appdirs.user_data_dir, 'hamster-gtk.sqlite'),
+            # Frontend
+            'autocomplete_activities_offset': 30,
```

Comment:
I do not like the name either, but fail to come up with somehting better.
I think ``activity_autocomplete_range`` is worthe than starting with "autocomplete" as the latter limits the "namespace" more than "activity" and hence already makes the name more specific towards its use. Also it is not used  to "autocomplete activities" but to "fetch activities for autocompletion in general". I agree though that ``..._range`` or maybe even better ``..._days`` would be clearer.